### DISCUSSION TITLE
0'm-Y g0T

### DISCUSSION THREAD

**Discussion by Azoahai** [IBAX-io/go-ibax#476] [Q&A] [answered]
### Clear and concise description of the problem

Everything is very clear and useful for ibax, dear administrators. we can't find fault. Can you give me a clue 🤔🤔🥁

### Suggested solution

Yes i listening sir😶

### Alternative

Eeee

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by HalloKyle** [comment, marked as answer]
Thank you for your support.
